app/uk/gov/hmrc/gform/auth/AuthService.scala

```diff
@@ -330,27 +328,22 @@ class AuthService(
     authResult match {
       case AuthSuccessful(AuthenticatedRetrievals(_, _, AffinityGroup.Individual, _, maybeNino, _, confidenceLevel), _)
           if maybeNino.isEmpty || confidenceLevel.level < Try(minimumCL.toInt).getOrElse(0) =>
+        logger.info(
+          s"Redirect to IV journey - nino: ${maybeNino.map(_ => "non-empty").getOrElse("empty")}, confidenceLevel: ${confidenceLevel.level}"
+        )
         val completionUrl = URLEncoder.encode(gform.routes.NewFormController.dashboard(formTemplate._id).url, "UTF-8")
         val failureUrl =
           URLEncoder.encode(gform.routes.IdentityVerificationController.failure(formTemplate._id).url, "UTF-8")
         AuthRedirect(
           s"/mdtp/uplift?origin=gForm&completionURL=$completionUrl&failureURL=$failureUrl&confidenceLevel=$minimumCL"
         )
-      case AuthSuccessful(AuthenticatedRetrievals(_, enrolments, AffinityGroup.Organisation, _, None, _, _), _) =>
-        val irsa = IRSA()
-        val maybeEnrolmentId: Option[EnrolmentIdentifier] =
-          enrolments.getEnrolment(irsa.name).flatMap(_.getIdentifier(irsa.id))
-
-        maybeEnrolmentId.fold[AuthResult](
-          AuthRedirect(
-            gform.routes.IdentityVerificationController.enrolmentsNeeded(formTemplate._id).url
-          )
-        )(_ => authResult)
-
+      case AuthSuccessful(AuthenticatedRetrievals(_, _, AffinityGroup.Organisation, _, _, _, _), _) =>
```

Comment:
Now we just block users that have organization affinity group. It was a bit more complex behavior here https://jira.tools.tax.service.gov.uk/browse/GFORMS-819 
Is that requirements no longer needed?